releng/config-forker/main_test.go

```diff
@@ -439,6 +439,39 @@ func TestGeneratePresubmits(t *testing.T) {
 					SkipBranches: []string{`release-\d\.\d`},
 				},
 			},
+			{
+				JobBase: config.JobBase{
+					Name:        "pull-kubernetes-e2e-branch-in-name-master",
+					Annotations: map[string]string{forkAnnotation: "true"},
+				},
+				Brancher: config.Brancher{
+					SkipBranches: []string{`release-\d\.\d`},
+				},
+			},
+			{
+				JobBase: config.JobBase{
+					Name:        "pull-kubernetes-e2e-keep-context",
+					Annotations: map[string]string{forkAnnotation: "true"},
+				},
+				Brancher: config.Brancher{
+					SkipBranches: []string{`release-\d\.\d`},
+				},
+				Reporter: config.Reporter{
+					Context: "mycontext",
+				},
+			},
+			{
+				JobBase: config.JobBase{
+					Name:        "pull-kubernetes-e2e-replace-context",
+					Annotations: map[string]string{forkAnnotation: "true"},
+				},
+				Brancher: config.Brancher{
+					SkipBranches: []string{`release-\d\.\d`},
```

Comment:
Hmm looks like `cmp` does not like the unexported fields in `config.Brancher` for comparison: https://github.com/kubernetes/test-infra/blob/882eb3f17e8e4f1344f7198ee161fb51ba471f2f/prow/config/jobs.go#L215

Do you want to add a custom comparator?

Reply:
On it. thanks.

Reply:
Done.